integration-tests/src/test/java/com/redhat/parodos/flows/SimpleRestartWorkFlowTest.java

```diff
@@ -332,7 +332,7 @@ public void runRestartWorkFlowComplexMultipleTimes() throws ApiException, Interr
 		assertEquals(WorkFlowStatusResponseDTO.StatusEnum.COMPLETED, workFlowStatusResponseDTO.getStatus());
 		log.info("Onboarding workflow execution completed with status {}", workFlowStatusResponseDTO.getStatus());
 		List<UUID> restartedIds = new ArrayList<>(5);
-		for (int i = 1; i <= 20; i++) {
+		for (int i = 1; i <= 10; i++) {
```

Comment:
@gabriel-farache WDYT?

Reply:
it seems OK but I am not sure it will solve the instability of the integrations tests